projects/migrations/0009_project_visibility.py

```diff
@@ -0,0 +1,50 @@
+# Generated by Django 5.1.4 on 2024-12-13 14:09
+
+from django.db import migrations, models
+from django.db.models import When, Case, Value, Q
+
+
+def migrate(apps, schema_editor):
```

Comment:
not a fan of using the name "migrate" here. I'd prefer a more detailed name like "set_visibility_field"